packages/stable/src/exports.gen.ts

```diff
@@ -8,12 +11,12 @@ export {
   AnnotationsBoundingVolume,
   AnnotationsBox,
   AnnotationsClassification,
-  AnnotationsCogniteAnnotationTypesDiagramsAssetLink,
-  AnnotationsCogniteAnnotationTypesDiagramsInstanceLink,
-  AnnotationsCogniteAnnotationTypesImagesAssetLink,
-  AnnotationsCogniteAnnotationTypesImagesInstanceLink,
-  AnnotationsCogniteAnnotationTypesPrimitivesGeometry2DGeometry,
-  AnnotationsCogniteAnnotationTypesPrimitivesGeometry3DGeometry,
+  AnnotationsCogmonoAnnotationTypesDiagramsAssetLink,
+  AnnotationsCogmonoAnnotationTypesDiagramsInstanceLink,
+  AnnotationsCogmonoAnnotationTypesImagesAssetLink,
+  AnnotationsCogmonoAnnotationTypesImagesInstanceLink,
+  AnnotationsCogmonoAnnotationTypesPrimitivesGeometry2DGeometry,
+  AnnotationsCogmonoAnnotationTypesPrimitivesGeometry3DGeometry,
```

Comment:
looks like a breaking change? any way possible to keep the old names?

Reply:
That's what annoys me the most, I don't have control over codegen and this is a change coming from the tool. How can I avoid this breaking change?

Reply:
My advice is not to use it since you spend more time working around it than it requires writing a few type definitions.

but maybe @andersfylling or @BugGambit have a better idea

Reply:
There is a discussion about this specific breaking change here: https://github.com/cognitedata/cognite-sdk-js/pull/1120#issuecomment-2209470020
I suggest:
1. Revert the breaking change.
2. Change the base branch to `release-v10` where we, in a few days, will release a new SDK major version where breaking changes are allowed.

Reply:
@polomani how can I keep the old names?